config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root controller: :restaurants, action: :index
   resources :restaurants
-  resources :carts
+  resources :carts do
+    get :remove_item
```

Comment:
Nice. Could we do that the the "add" method too? I tried something very similar but couldn't get it to work. Thomas told me to leave it though so maybe we shouldn't try it.

Reply:
we could probably do that, might need to change a couple of things, but that should be done in another PR.